src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -5010,14 +5010,19 @@ private StatementAst UsingStatementRule(Token usingToken)
                     SkipToken();
 
                     var aliasToken = NextToken();
-                    if (aliasToken.Kind == TokenKind.EndOfInput)
+                    if (aliasToken.Kind is TokenKind.EndOfInput or TokenKind.NewLine or TokenKind.Semi)
                     {
                         UngetToken(aliasToken);
                         ReportIncompleteInput(After(equalsToken),
                             nameof(ParserStrings.MissingNamespaceAlias),
                             ParserStrings.MissingNamespaceAlias);
                         return new ErrorStatementAst(ExtentOf(usingToken, equalsToken));
                     }
+                    if (aliasToken.Kind == TokenKind.Comma)
```

Comment:
Please add blank line.
```suggestion

                    if (aliasToken.Kind == TokenKind.Comma)
```